### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized control type property is wrong for "Search Here" button

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#47]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/VSCode in EDGE
2. Navigate to the Search button available beside the "Sign In" link at the top right corner of the page
3. Observe the "Localized Control Type" property in Inspect

**Same issue is also observed with the link:**
https://marketplace.visualstudio.com/items?itemName=donjayamanne.python

**Actual Result:**
The "Localized Control Type" property is "Text"

**Narrator Behaviour:**
narrator announces as "Search Here"

**Expected Result:**
The "Localized Control Type" property should be button
Eg: "Search Here" Button

**User Impact:**
Visually Impaired users will not be able to understand and will be confused about what control type the search is.

**Screenshots:**
![localized control type of search button is text](https://cloud.githubusercontent.com/assets/24825441/21718212/f933085a-d43b-11e6-83db-0cf46d5829ac.png)

**Comment by waderyan** [comment]
Marketplace

**Comment by waderyan** [comment]
Please send to Marketplace team.
